lib/metadata/acl.js

```diff
@@ -12,11 +12,12 @@ const acl = {
         metadata.updateBucket(bucket.getName(), bucket, log, cb);
     },
 
-    addObjectACL(bucket, objectKey, objectMD, addACLParams, log, cb) {
+    addObjectACL(bucket, objectKey, objectMD, addACLParams, log, cb, params) {
```

Comment:
would it be possible to keep the cb as the last argument throughout this pr?  so params would come after addACLParmas